README.md

```diff
@@ -1,6 +1,6 @@
-# Google Software Product Sprint
+# Yusuf's SPS Portfolio
 
-This repo contains the projects you'll work on throughout SPS.
+This repo contains the projects I'll work on throughout SPS.
 
 To get started:
```

Comment:
Can probably remove this "To get started" section since it doesn't apply to people viewing your portfolio repo (it's an instruction for yourself to set up this portfolio)